airframe-config/src/main/scala/wvlet/airframe/config/package.scala

```diff
@@ -42,16 +42,24 @@ package object config {
     }
 
     def bootstrapWithConfigProcessor(configProcessor: Config => Unit): Design = {
-      configProcessor(getConfig)
+      configProcessor(getConfigOrEmpty)
       d
     }
 
-    private def getConfig: Config = {
+    // Get config binded in the design.
+    def getConfig: Option[Config] = {
       d.getDesignConfig match {
         case dc: DesignOptionsWithConfig =>
-          dc.config
+          Some(dc.config)
         case _ =>
-          Config()
+          None
+      }
+    }
+
+    def getConfigOrEmpty: Config = {
```

Comment:
`currentConfig` might be better to distinguish with Option[Config]

Reply:
👍